modules/internal/ExportWrappers.chpl

```diff
@@ -24,6 +24,31 @@ module ExportWrappers {
   use CPtr;
   use SysCTypes;
 
+  private proc _initDynamicEndCount() {
+    var endCount = _endCountAlloc(false);
```

Comment:
Can you do `forceLocalTypes=false` (maybe a personal preference, but I like named arguments for bool arguements)

Reply:
That's a good personal preference!